Dockerfile

```diff
@@ -17,16 +17,20 @@ RUN mamba env create -n raven -f environment.yml \
 # Add the raven conda environment to the path
 ENV PATH /opt/conda/envs/raven/bin:$PATH
 
+# Set bind address and port from env variable of parent system
+ENV RAVEN_BIND_ADDRESS=${RAVEN_BIND_ADDRESS:-0.0.0.0}
+ENV RAVEN_BIND_PORT=${RAVEN_BIND_PORT:-9099}
+
 # Copy raven source code
 COPY . /code
 
 # Install raven
 RUN pip install . --no-deps
 
-# Start WPS service on port 9099 on 0.0.0.0
-EXPOSE 9099
+# Start WPS service on port 9099
+EXPOSE $RAVEN_BIND_PORT
 
-CMD ["gunicorn", "--bind=0.0.0.0:9099", "raven.wsgi:application"]
+CMD ["gunicorn", "--bind=$RAVEN_BIND_ADDRESS:$RAVEN_BIND_PORT", "raven.wsgi:application"]
```

Comment:
I am pretty sure `$<env>` format does not work in `CMD`. 
Not sure about `EXPOSE`. 

The workaround is to use an entrypoint shell script, which then uses the variables.

Reply:
One easy way to confirm env var works in `CMD` and `EXPOSE` is to build the docker image and test it in the PAVICS stack.

Reply:
I did that and as I remembered, docker run raises something related to `$RAVEN_BIND_ADDRESS:$RAVEN_BIND_PORT` being invalid because `$RAVEN_BIND_PORT` doesn't get replaced by the provide integer.

Reply:
Given `ENV RAVEN_BIND_PORT=${RAVEN_BIND_PORT:-9099}`, if we can make the launching command "see" the env var, it should work.  So probably use `ENTRYPOINT` instead of `CMD`.  If needed, force starting a new shell to "see" the env var with `bash -c "gunicorn --bind=$RAVEN_BIND_ADDRESS:$RAVEN_BIND_PORT raven.wsgi:application"`.

Or research a recipe for how to properly start gunicorn inside a Docker image.